### PR TITLE
APII-3552: Updated LICENSE

### PR DESCRIPTION
**Pull request by brijendra98**
Updated LICENCE to Apache 2.0 .

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Copyright 2016-2017 Bazaarvoice, Inc.
```
Comment (arnm) [inline LICENSE]: update year to 2018

Comment (brijendra98) [reply LICENSE]: Changed

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Copyright 2016 Bazaarvoice, Inc.
```
Comment (mmanguno) [inline LICENSE]: 2018

Comment (mmanguno) [reply LICENSE]: nm

Comment (brijendra98) [reply LICENSE]: https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated

**Pull request merged**